src/Features/Core/Portable/ConvertLinq/ConvertForEachToLinqQuery/IConverter.cs

```diff
@@ -9,6 +9,6 @@ internal interface IConverter<TForEachStatement, TStatement>
     {
         ForEachInfo<TForEachStatement, TStatement> ForEachInfo { get; }
 
-        void Convert(SyntaxEditor editor, CancellationToken cancellationToken);
+        void Convert(SyntaxEditor editor, bool convertToQuery, CancellationToken cancellationToken);
```

Comment:
>convertToQuery [](start = 47, length = 14)

I hope we will have just two options. However, enum seems to be better here.

Reply:
I am not sure if we have more options. We can switch to enum once we have another option.